portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,153 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adhya Nagpal</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id="name">Adhya Nagpal</h1>
+      <br>
+      <img src="../public/AdhyaNagpal.png" width="300" height="300">
+      <!-- <p>Click here to get a random message:</p>
+      <button onclick="addRandomGreeting()">Hello</button> -->
       <div id="greeting-container"></div>
     </div>
+
+    <div id="intro">
+        <p>Hey there! I'm Adhya. I was born and brought up in Delhi. I'm a coding enthusiast and love to read fiction, play badminton and listen to music in my free time. I've participated in a couple of hackathons like HackIIITD'19 and UHack 4.0 and some open source events like Hacktoberfest'19.</p>
```

Comment:
This line is very large, break it into multiple lines.

Reply:
Okay, will correct this.